test/index.js

```diff
@@ -2,88 +2,20 @@
 
 const Net = require('net');
 
-const Code = require('@hapi/code');
+const IoRedis = require('ioredis');
 const Catbox = require('@hapi/catbox');
-const CatboxRedis = require('..');
-const Hapi = require('@hapi/hapi');
 const Hoek = require('@hapi/hoek');
-const IoRedis = require('ioredis');
-const Lab = require('@hapi/lab');
-
 const Mock = require('./mock');
+const { Engine: CatboxRedis } = require('..');
 
+const Code = require('@hapi/code');
+const Lab = require('@hapi/lab');
 
 const internals = {};
 
-
 const { it, describe } = exports.lab = Lab.script();
 const expect = Code.expect;
 
-
-describe('hapi', () => {
```

Comment:
I'm curious, what was the reason behind deleting all these tests?

Reply:
Ah good question, thanks for bringing it up.  We can totally put them back if it's preferable.  There are a few reasons I pulled them out, but none of them are very important:
 - There was code in there specific to hapi v17.
 - None of the other catbox engines use hapi in their test suites, so this was an outlier.
 - I couldn't tell what it was actually testing—I think it may have been written as a smoke test to check progress during the big refactor that Eran did a few years ago.

Reply:
Alright, thanks for the explanation. We keep them off then.